presto-spark-base/src/main/java/com/facebook/presto/spark/execution/PrestoSparkTaskExecutorFactory.java

```diff
@@ -407,8 +407,17 @@ public <T extends PrestoSparkTaskOutput> IPrestoSparkTaskExecutor<T> doCreate(
         TaskId taskId = new TaskId(new StageExecutionId(stageId, 0), partitionId);
 
         // TODO: Remove this once we can display the plan on Spark UI.
-
-        log.info(PlanPrinter.textPlanFragment(fragment, functionAndTypeManager, session, true));
+        // Currently, `textPlanFragment` throws an exception if json-based UDFs are used in the query, which can only
```

Comment:
is this specific to json-based udfs or any custom function namespace manager (e.g. mysql)

Reply:
I am not 100% sure of the answer. According to my tests, I only saw this in JSON-based UDFs.